### PR TITLE
Streams

### PR DESCRIPTION
**Pull request by jardmclan**
This branch adds the Streams service to the repository and updates the openapi spec to the latest version.

Issue #19

### CODE REVIEW

File: example/src/streams.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
+
+// Load e2e environment variables into process.env
+require('dotenv').config({ path: 'example.env' });
+const fetch = require('node-fetch');
+import { getToken, checkJsonError } from '../../tapis-typescript/e2e/utils';
+import { Authenticator, Streams } from '@tapis/tapis-typescript';
+
+const version = 5;
```
Comment (jchuahtacc) [inline example/src/streams.ts:8]: What does this version number refer to?

File: example/src/streams.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
+
+// Load e2e environment variables into process.env
+require('dotenv').config({ path: 'example.env' });
```
Comment (jchuahtacc) [inline example/src/streams.ts:3]: Is there an updated example.env? Should that include the constants for E2E_TEST_...?

File: tapis-typescript/e2e/streams.spec.ts [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+} from '../src';
+import { getToken, checkJsonError } from './utils';
+import { expect } from 'chai';
+import fetch from 'cross-fetch';
+
+const version = 5;
+
+const projectId = `E2E_TEST_PROJECT_${version}`;
+const siteId = `E2E_TEST_SITE_${version}`;
+const instId = `E2E_TEST_INSTRUMENT_${version}`;
```
Comment (jchuahtacc) [inline tapis-typescript/e2e/streams.spec.ts:17]: These should be retrieved from process.env with the values stored in e2e.env.

**Review by jchuahtacc — CHANGES_REQUESTED** [review]

File: example/src/streams.ts (hunk 1 above)
Comment (jardmclan) [reply example/src/streams.ts:8]: I went through multiple versions of the test objects while testing the spec, I was using the version number as an easy way to update this.

Comment (jardmclan) [reply example/src/streams.ts]: migrated to env

File: example/src/streams.ts (hunk 2 above)
Comment (jardmclan) [reply example/src/streams.ts:3]: Updated example.env and migrated ids to this.

File: tapis-typescript/e2e/streams.spec.ts (hunk 3 above)
Comment (jardmclan) [reply tapis-typescript/e2e/streams.spec.ts]: Migrated ids to e2e.env (listed in e2e.env.sample.

**Review by nathandf — APPROVED** [review]

**Review by jchuahtacc — APPROVED** [review]

**Pull request merged**